bin/server.js

```diff
@@ -4,7 +4,9 @@ const path = require('path');
 
 const chalk = require('chalk');
 const chokidar = require('chokidar');
-require('babel-register');
+require('babel-register')({
+  plugins: ['dynamic-import-node'],
```

Comment:
Can't we enable it globally in the `.babelrc` file?

Reply:
If we do that it would be transpiled for the client when it doesn't need to be, because webpack understands `import()`.